docs/Makefile

```diff
@@ -34,12 +34,14 @@ resources.mk: sources.csv resourceContent.mk.tmpl
 	@echo >> $@
 	@for line in $$(cat sources.csv); do \
     sed -e "s|@@NAME|$$(echo $$line | cut -d, -f1 | tr -d '\n')|g" \
-        -e "s|@@SLUG|$$(echo $$line | cut -d, -f1 | tr -d '\n' | sed 's/\./-/g')|g" \
-        -e "s|@@SOURCE|$$(echo $$line | cut -d, -f2 | tr -d '\n')|g" \
-        -e "s|@@EXAMPLE|$$(echo $$line | cut -d, -f3 | tr -d '\n')|g" \
-        -e "s|@@TYPE|$$(echo $$line | cut -d, -f4 | tr -d '\n')|g" \
-        -e "s|@@TARGET|content/resources/$$(echo $$line | cut -d, -f1 | tr -d '\n').md|g" \
-        resourceContent.mk.tmpl >> $@; \
+	-e "s|@@SLUG|$$(echo $$line | cut -d, -f1 | tr -d '\n' | sed 's/\./-/g')|g" \
```

Comment:
The spaces were replaced with tabs for consistency with other parts of the makefile (and my editor having save-hooks that does it automatically for makefiles)